app/views/crops/_predictions.html.haml

```diff
@@ -28,31 +28,26 @@
     .card.fact-card
       .card-body.text-center
         %h3 Median lifespan
-        %strong= crop.median_lifespan
+        %strong= crop.median_lifespan / 7
         %i.fas.fa-sun-o.fa-5x.pt-3.amber-text
-        %span days
+        %span #{'week'.pluralize(crop.median_lifespan / 7)}
```

Comment:
Worth introducing a 'in_weeks' helper for this?

Reply:
ooooh! thanks. I'll use that

Reply:
i can't find this. 
`undefined method `in_weeks' for 78:Integer`

Can you point me at the docs?

Reply:
we only want the word "weeks" here, not the number. The number is passed so it works out if it needs to be plural